test/spanningtrees/prim.jl

```diff
@@ -0,0 +1,75 @@
+using LightGraphs
+using Base.Test
+
+g = CompleteGraph(4)
+
+distmx = [
+  0  1  5  6
+  1  0  4  10
+  5  4  0  3
+  6  10  3  0
+]
+
+# Testing Prim's algorithm
+mst = prim_mst(g, distmx)
+vec_mst = Vector{Edge}([Edge(1, 2), Edge(2, 3), Edge(3, 4)])
+
+@test mst == vec_mst
+
+#second test
+g = Graph(8)
+add_edge!(g, 2,8)
+add_edge!(g, 1,3)
+add_edge!(g, 6,8)
+add_edge!(g, 1,8)
+add_edge!(g, 3,4)
+add_edge!(g, 2,4)
+add_edge!(g, 2,6)
+add_edge!(g, 3,8)
+add_edge!(g, 6,5)
+add_edge!(g, 2,3)
+add_edge!(g, 5,7)
+add_edge!(g, 1,7)
+add_edge!(g, 4,7)
+add_edge!(g, 7,3)
+add_edge!(g, 1,5)
+add_edge!(g, 5,8)
+
+distmx_sec = zeros(8, 8)
+distmx_sec[2,8] = 0.19
+distmx_sec[8,2] = 0.19
```

Comment:
if you don't want to do the matrix as you did above, you can always do `a = b = 5`